Makefile.am

```diff
@@ -138,16 +140,26 @@ check_PROGRAMS = test/run-tests
 if OS390
 test_run_tests_CFLAGS =
 else
+if GCC
 test_run_tests_CFLAGS = -Wno-long-long
+else
+test_run_tests_CFLAGS =
+endif
 endif
 
 if SUNOS
+if GCC
 # Can't be turned into a CC_CHECK_CFLAGS in configure.ac, it makes compilers
 # on other platforms complain that the argument is unused during compilation.
 test_run_tests_CFLAGS += -pthreads
 endif
+endif
 
+if SUNOS
+test_run_tests_LDFLAGS = -lm
```

Comment:
I'd be okay with just adding `-lm` unconditionally. If it's just for the floor() call in test-loop-handles.c, I'd replace that with truncating integer division to avoid the dependency on libm.

Reply:
Ok, I will remove the call to `floor()`.